.golangci-kal.yml

```diff
@@ -58,33 +58,21 @@ issues:
   max-issues-per-linter: 0
   exclude-rules:
   # KAL should only run on API folders.
-  - path-except: "api/*"
-    linters:
-      - kal
-  # apiwarnings contains a `api` characters in the path, but it's not an API folder.
-  - path: "util/apiwarnings/*"
+  - path-except: "api//*"
     linters:
       - kal
-  - path: "api/v1beta1/*|api/v1alpha1/*"
+  - path: "api//*"
```

Comment:
This one still wants to be v1beta1 so that when we introduce new v1beta2 APIs this is enforced